src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball[] balls = new Ball[3];
+        for (Ball ball: balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
there should be a space before and after semicolon : , use idea's formatter - Ctrl Alt L
try to rewrite this solution without using array